pyfr/writers/paraview.py

```diff
@@ -277,7 +276,7 @@ def subnodes(cls, n):
             conbase = np.hstack((conbase, conbase + (1 + n)**2))
 
         # Calculate offset of each subdivided element's nodes
-        nodeoff = np.zeros((n,)*cls.ndim)
+        nodeoff = np.zeros((n,)*cls.ndim, dtype='int')
```

Comment:
`dtype=np.int32` or `dtype=np.int` or `dtype=np.int64` (whatever is most appropriate.